CHANGELOG.md

```diff
@@ -17,32 +26,29 @@
   subscripted with an `Unpack` object.
   Patch by [Daraan](https://github.com/Daraan).
 - Backport to Python 3.10 the ability to substitute `...` in generic `Callable`
-aliases that have a `Concatenate` special form as their argument.
+  aliases that have a `Concatenate` special form as their argument.
   Patch by [Daraan](https://github.com/Daraan).
 - Extended the `Concatenate` backport for Python 3.8-3.10 to now accept
   `Ellipsis` as an argument. Patch by [Daraan](https://github.com/Daraan).
 - Fix backport of `get_type_hints` to reflect Python 3.11+ behavior which does not add
   `Union[..., NoneType]` to annotations that have a `None` default value anymore.
   This fixes wrapping of `Annotated` in an unwanted `Optional` in such cases.
   Patch by [Daraan](https://github.com/Daraan).
-- Fix error in subscription of `Unpack` aliases causing nested Unpacks 
+- Fix error in subscription of `Unpack` aliases causing nested Unpacks
```

Comment:
Nitty nit: 
```suggestion
- Fix error in subscription of `Unpack` aliases causing nested Unpacks.
```

Reply:
Sorry I didn't see this before hitting merge. I can do a fixup PR after I cut the release